public/static/locales/ru/common.json

```
@@ -175,6 +175,7 @@
   "welcome_to_calcom": "Добро пожаловать в Cal.com",
   "welcome_instructions": "Скажите как вас зовут и в каком вы часовом поясе. Эту информацию потом можно будет изменить.",
   "connect_caldav": "Подключиться к серверу CalDav",
+  "connect_exchnage": "Подключиться к серверу Exchnage",
```

```suggestion
  "connect_exchange": "Подключиться к серверу Exchnage",
```
Exchange is misspelled here :)